meson.build

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
 project('lxcfs', 'c',
-        version : '4.0.0',
+        version : '4.0.9',
```

Comment:
Don't do this, the `4.0.9` version should only be in the `stable-4.0` branch. Master remains at `4.0.0` until we release a `4.1.0` or a `5.0.0`.